isis/src/galileo/apps/gllssi2isis/main.cpp

```diff
@@ -102,35 +94,34 @@ void IsisMain() {
   Pvl pdsLabel;
   p.SetPdsFile(inFile.expanded(), "", pdsLabel);
 
-  //Set up the output file
-  Cube *ocube;
-
+  // If summed handle the image similarly to pds2isis
+  // with an extra translation step
   if (!summed) {
-    ocube = p.SetOutputCube("TO");
+    Cube *ocube = p.SetOutputCube("TO");
```

Comment:
Does ProcessImport pass ownership or does it delete this when EndProcess is called?